### DISCUSSION TITLE
phân công bài thực hành word

### DISCUSSION THREAD

**Discussion by TranBaoLam10C13** [dangxuantruong10a13/nhom9_10a13#26] [General]
làm gì mọi người

**Comment by dangxuantruong10a13** [comment]
Làm j ta õmg

**Comment by dangxuantruong10a13** [comment]
@04lananh @NgHan12pn @TangVinhAn-PNc13 @tranphamkhuongduy10a13pn

**Comment by dangxuantruong10a13** [comment]
Giờ nè ãe hãy hãy hãy hãy hãy giải cứu diễm my à lộn aisshhh chết tiệt. Mn đi đọc lại cái notion đi cho 5p é

**Comment by dangxuantruong10a13** [comment]
https://hiencaophunhuan.notion.site/

**Comment by dangxuantruong10a13** [comment]
Link cho 2 pn mới, qươn đưa chứ, thiệt là đáng xấu hổ

**Comment by 04lananh** [comment]
mng ơi h chọn chủ đề trước ik

**Reply by dangxuantruong10a13** [reply]
Chủ đề j ta @tranphamkhuongduy10a13pn @TranBaoLam10C13 @NgHan12pn @TangVinhAn-PNc13

**Reply by TranBaoLam10C13** [reply]
Sử dụng thành ngữ / tục ngữ / ca dao hay thơ tự sáng tác để bày tỏ quan điểm, nghị luận về
cuộc sống xung quanh mà bản thân cảm nhận được

**Reply by 04lananh** [reply]
thầy bảo là chọn một chủ đề để viết á

**Reply by TranBaoLam10C13** [reply]
chắc nói về cuộc sống

**Reply by dangxuantruong10a13** [reply]
Chủ đề alime wibu đi mn
